source/_integrations/flipr.markdown

```diff
@@ -11,15 +11,35 @@ ha_codeowners:
 ha_domain: flipr
 ha_platforms:
   - binary_sensor
+  - select
   - sensor
+  - switch
 ha_integration_type: integration
 ---
 
-[Flipr](https://www.goflipr.com) is a smart pool monitor that publishes data to the cloud via SigFox.
+[Go flipr](https://www.goflipr.com) company sells smart pool monitor and management devices. The Flipr and Flipr Hub devices publish data to the cloud via Wi-Fi and SigFox.
+This {% term integration %} gives you access on Home Assistant to the information measured by your Flipr with the same data as the vendor's smartphone application.
+This {% term integration %} gives you also access to the Flipr Hub to control your pool equipments like pump, heater, light, etc.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Flipr](#flipr)
+- [Hub](#hub)
+
+## Prerequisites
+
+You will need to use the standalone app for this device to register a username and password.
+
+- [Google](https://play.google.com/store/apps/details?id=com.goflipr.flipr)
+- [Apple](https://apps.apple.com/fr/app/flipr/id1225898851)
+
+{% include integrations/config_flow.md %}
+
+## Flipr
+
 Flipr sends data like pH, chlorine or temperature to a cloud server on a regular basis in order to monitor your pool.
-This integration gives you access to this information on Home Assistant in addition to the vendor's smartphone application.
 
-There is currently support for the following information within Home Assistant:
+There is currently support for the following information within Home Assistant via **sensors** and **binary_sensors** :
```

Comment:
Could you add a section called ## Entities, and below that a separate section for each type, ### Sensors, ### Binary sensors, etc and list them with name and description?

Reply:
yes, thank you! Could you also give the units, where applicable?